.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,28 @@
+<!-- 
+Insert issue number that this PR fixes (if any) just after 'Fixes #'.
+If this PR does not fix an existing issue, consider opening one or remove 'Fixes #' from the PR description.
+-->
 Fixes #
 
 Changes made in this Pull Request:
- - 
+<!-- Describe the changes that this PR makes. If applicable, use the following bullet list. --> 
+-
 
-
-PR Checklist
-------------
- - [ ] Tests?
- - [ ] Docs?
- - [ ] CHANGELOG updated?
+## PR Checklist
+<!-- Please use the following checklist to ensure the PR is ready to be reviewed/merged. -->
  - [ ] Issue raised/referenced?
+ - [ ] Tests updated/added?
+ - [ ] Documentation updated/added?
+ - [ ] `package/CHANGELOG` file updated?
+ - [ ] Is your name in `package/AUTHORS`? (If it is not, add it!)
+
+## Developers Certificate of Origin
+<!-- 
+CHECKING THE FOLLOWING BOX (BY ADDING 'x' INSIDE THE BRACKETS `[x]`) IS MANDATORY FOR ALL PRs.
```

Comment:
So as per my last comment, I'd be perfectly happy with this being there without a tick box - especially if it's going to be a pain for others.

That is to say, if you wanted to just make this an implicit agreement - i.e. a footer that says "by summiting this code you agree to.." then that'd have the same effect. At the end of the day the good actors will act well and the bad actors will act badly.

Reply:
Happy to go with a non-ticking implicit agreement. The comment can then say to not delete/modify the following lines one wants the PR to be merged?